index.js

```diff
@@ -6,6 +6,9 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module'
   },
+  overrides: [{
+    files: ['*.cjs', '*.mjs'],
```

Comment:
We might want to add `*.vue` also.

Reply:
Why?  We use `extends: [..., 'digitalbazaar/vue', ...]` in our vue projects which should handle all `.vue` files.  If that's not working, open a different issue.